invenio_rdm_records/services/permissions.py

```diff
@@ -214,6 +214,23 @@ class RDMRecordPermissionPolicy(RecordPermissionPolicy):
     can_draft_media_update_files = can_review
     can_draft_media_delete_files = can_review
 
+    #
+    # Media files - record
+    #
+    can_read_media_files = [
+        IfRestricted("record", then_=can_view, else_=can_all),
+        ResourceAccessToken("read"),
+    ]
+    can_get_content_media_files = [
+        # note: even though this is closer to business logic than permissions,
+        # it was simpler and less coupling to implement this as permission check
+        IfFileIsLocal(then_=can_read, else_=[SystemProcess()])
+    ]
+    can_create_media_files = [Disable()]
+    can_set_content_media_files = [Disable()]
+    can_commit_media_files = [Disable()]
+    can_update_media_files = [Disable()]
+    can_delete_media_files = [Disable()]
```

Comment:
@slint sanity check: if we disable `can_create_media_files` shouldn't be enough to say "users by default cannot create in any manner media files"?

Reply:
- Shouldn't it be `can_draft_media_create_files = [Disable()]` (and all other `can_draft_media_...` vars), since that's how users actually get to create the media files?
- The naming prefixes are a bit mixed actually, shouldn't it be `can_draft_media_files_create`? (note the last two words are flipped)